geoserver/v0.1.0/README.md

```diff
@@ -0,0 +1,126 @@
+# GeoServer Helm Chart
+
+## Helm Chart Installation
+
+1) Clone the helm chart repo
+
+```bash
+git clone https://github.com/geosolutions-it/charts.git
+```
+
+2) Within the root of the helm chart, create a custom `values.yaml` file
+
+```bash
+cp geoserver/v0.1.0/values.yaml geoserver/v0.1.0/custom-values.yaml
+```
+
+3) Edit sample `context.xml` to add as many JNDI datasources are needed. The sample `context.xml` contains by default the JNDI datasource for the embedded postgis service that can be enabled in `values.yaml`. Example:
+
+```xml
+<Context>
+
+    <!-- Default set of monitored resources -->
+    <WatchedResource>WEB-INF/web.xml</WatchedResource>
+
+    <!-- Uncomment this to disable session persistence across Tomcat restarts -->
+    <!--
+    <Manager pathname="" />
+    -->
+
+    <!-- Uncomment this to enable Comet connection tacking (provides events
+         on session expiration as well as webapp lifecycle) -->
+    <!--
+    <Valve className="org.apache.catalina.valves.CometConnectionManagerValve" />
+    -->
+
+    <Resource
+     name="jdbc/postgres" auth="Container" type="javax.sql.DataSource"
+     driverClassName="org.postgresql.Driver"
+     url="jdbc:postgresql://127.0.0.1:5432/geoserver"
+     username="geoserver" password="geoserver"
+     initialSize="0" maxActive="20" maxIdle="5" maxWait="2000" minIdle="0"
+     timeBetweenEvictionRunsMillis="30000"
+     minEvictableIdleTimeMillis="60000"
+     validationQuery="SELECT 1"
+     maxAge="600000"
+     rollbackOnReturn="true"
+     />
+</Context>
+```
+
+Concerning the security context of pod and pv(s), it is recommended not to comment out this section, because the docker image runs natively as non root, althogh the id or gid of `1000` can be changed to anything else and they can be differnt values, in case the infrastructure requires different ID/GID mapping:
+
+```yml
+podSecurityContext:
+  fsGroup: 1000
+
+securityContext:
+  runAsNonRoot: true
+  runAsUser: 1000
+```
```

Comment:
```suggestion
Concerning the security context of pod and pv(s), it is recommended not to comment out this section, because the docker image runs natively as non root, althogh the id or gid of `1000` can be changed to anything else and they can be differnt values, in case the infrastructure requires different ID/GID mapping:

```yml
podSecurityContext:
  fsGroup: 1000

securityContext:
  runAsNonRoot: true
  runAsUser: 1000
```
```

Remove this